.github/workflows/test.yml

```diff
@@ -9,6 +9,25 @@ on:
     types: [test-command]
 
 jobs:
+  foobar:
+    runs-on: ubuntu-latest
+    steps:
+    # Update GitHub status for dispatch events
+    - name: "Update GitHub Status for this ref"
+      #if: ${{ github.event_name == 'repository_dispatch' }}
+      uses: 'docker://cloudposse/github-status-updater'
+      with:
+        # We need to use args because GitHub actions overwrites many `GITHUB_*` environment variables
+        args: "-action update_state -ref ${{ github.event.pull_request.head.sha }}"
+      env:
+        GITHUB_TOKEN: ${{ secrets.PUBLIC_REPO_ACCESS_TOKEN }}
+        GITHUB_STATE: success
```

Comment:
```suggestion
        GITHUB_STATE: pending
```

Reply:
Oh, I'm just experimenting in this - too slow to wait for terratest to finish each time